fixtures/flows/brief/specialist.js

```diff
@@ -0,0 +1,129 @@
+export const create = async (params) => {
+    console.log(`Starting to create ${params.areaOfExpertise} brief`);
+    await selectLot('digital-professionals');
+    await createBrief();
+    await fillRole(params.title);
+    await selectLocation(params.locations);
+    await fillDescriptinoOfWork();
```

Comment:
Same typo here and in function definition.

Reply:
fixed